### PR TITLE
sorting algorithms

### PR DESCRIPTION
**Pull request by 13muskanp**
quick sort and merge sort

### CODE REVIEW

File: mergeSort.cpp [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+		cin>>arr[i];
+	}
+
+	merge_sort(arr, 0, N-1);
+	
+	cout<<endl<<"Output: ";
+	for(int i=0; i<N; i++)
+		cout<<arr[i]<<" ";	
+	cout<<endl;
+}
```
Comment (t0xic0der) [inline mergeSort.cpp:81]: Nice touch with the space-separated printing of list.

File: quickSort.cpp [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	{
+		cout<<"Enter "<<i+1<<"th element: ";
+		cin>>arr[i];
+	}
+	int x = 0;
+	int y = n-1;
+	quick_sort(arr, x, y);
+	cout<<endl;
+	for(int i=0; i<n; i++)
+		cout<<arr[i];
```
Comment (t0xic0der) [inline quickSort.cpp:59]: You would want to add separators between the integers when printing them out.

**Review by t0xic0der — CHANGES_REQUESTED** [review]
Make the suggested change in `quickSort.cpp` and then your PR should be ready for a merge.

File: quickSort.cpp (hunk 2 above)
Comment (ankitapuri) [reply quickSort.cpp:59]: you might make the changes

**Review by 13muskanp — COMMENTED** [review]
Hacktober accepted

**Review by ankitapuri — APPROVED** [review]

**Pull request merged**